src/App.tsx

```diff
@@ -5,21 +5,22 @@ import { Routes, Route, Link } from 'react-router-dom'
 import { CollectionProvider } from 'hooks'
 import { QueryClient, QueryClientProvider } from 'react-query'
 import { ReactQueryDevtools } from 'react-query/devtools'
+import { StyledNav, StyledNavLink } from './styles'
```

Comment:
all styled-components should be defined in the component files they're used in (under the default export)